WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationView.swift

```diff
@@ -67,3 +67,24 @@ struct TapToPayEducationView: View {
         })
     }
 }
+
+// MARK: - Hosting Controller
+
+final class TapToPayEducationViewViewHostingController: UIHostingController<TapToPayEducationView> {
```

Comment:
super nit: maybe just one `View` in the name?
```suggestion
final class TapToPayEducationViewHostingController: UIHostingController<TapToPayEducationView> {
```